src/clients/client.ts

```diff
@@ -188,7 +182,12 @@ class Client {
    */
   async swapIndexes(params: SwapIndexesParams): Promise<EnqueuedTask> {
     const url = "/swap-indexes";
-    return await this.httpRequest.post(url, params);
+    const taks = (await this.httpRequest.post({
+      relativeURL: url,
+      body: params,
+    })) as EnqueuedTaskObject;
+
+    return new EnqueuedTask(taks);
   }
```

Comment:
bug: previously this did not return `EnqueuedTask`, but rather `EnqueuedTaskObject`